site/guides/enhanced-apis/nft.md

```diff
@@ -3,30 +3,30 @@ outline: deep
 head:
   - - meta
     - property: og:title
-      content: How to Fetch a Smart Account's NFTs
+      content: How to Fetch a smart account's NFTs
```

Comment:
keep Smart Account capital-case on the metatags and section headers in this doc

Reply:
Done!

Kept capital-case in the titles and section headers but not in descriptions as everything is lower-case in descriptions.